### PR TITLE
Fix home screen content scrolling

### PR DESCRIPTION
**Pull request by henrytao-me**
This PR fixes scrolling in home screen content.

## How to test?
- Verify the home screen can scroll normally in both landscape and portrait. 
- Verify accessibility in home screen works.

### CODE REVIEW

File: src/screens/home/components/BaseHomeView.tsx [hunk 1]
```diff
@@ -32,18 +32,13 @@ export const BaseHomeView = ({children, iconName}: BaseHomeViewProps) => {
 
 const styles = StyleSheet.create({
   primaryIcon: {marginLeft: -40, marginBottom: 30},
-  flex: {
-    flex: 0,
```
Comment (henrytao-me) [inline src/screens/home/components/BaseHomeView.tsx:36]: Not sure why we use `flex: 0`. I don't think it's appropriate.

**Comment by timarney** [comment]
@smcmurtry please review

**Comment by henrytao-me** [comment]
@timarney Feel free to merge after you have a look 🙇

**Comment by smcmurtry** [comment]
Here is another before and after on Android, after scrolling all the way down.

Before:
Screen Shot 2020-07-27 at 10 00 18 AM

After:
Screen Shot 2020-07-27 at 9 59 57 AM

**Comment by smcmurtry** [comment]
Android EN framework off:
After:
Screen Shot 2020-07-27 at 9 58 30 AM

**Comment by smcmurtry** [comment]
Looks like landscape mode text is getting cut off on many screens in master, and this PR fixes that.

Before, scrolled all the way down:
Screen Shot 2020-07-27 at 10 08 44 AM

After:
Screen Shot 2020-07-27 at 10 08 30 AM

**Pull request merged**